SU-SWS/decanter
# Add linear gradient background mixin

# READY FOR REVIEW

# Summary
- Adding a linear gradient mixin (written by author in below link) 
https://www.sitepoint.com/building-linear-gradient-mixin-sass/

# Needed By (Date)
- Whenever

# Urgency
- not urgent

# Steps to Test

1. pull this branch
1. Use this mixin on a component in the styleguide, e.g., 
`@include linear-gradient(42deg, #b58234 0%, #d2b545 50%, #d7c04d 50.01%, $color-white 100%);`
2. Run` grunt styleguide` and see if it renders the expected gradient

# Affected Projects or Products
- Decanter
- Scott's event calendar

# Associated Issues and/or People
- @stocker
